hangman.rb

```diff
@@ -0,0 +1,18 @@
+# Hardcode solutionds
+def hangman(word, guess)
+  word.gsub(/[^#{guess.join('')}]/, "_")
+end
+
+puts hangman('hangman', ['h', 'g'])
+
+# Interactive solution
+def hangman(word)
+  guess = []
```

Comment:
Typically an array is going to be multiple things so it would be good to pluralize any variable that is an array. In this case would be `guesses = []`. That makes it easier to tell this could be multiple things just by looking at the name, rather than assuming it is only one thing.

Reply:
Thank you. Implemented.